jira-test/jira-test-docker/src/test/it/org/xwiki/contrib/jira/test/ui/PastePluginIT.java

```diff
@@ -0,0 +1,170 @@
+/*
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+package org.xwiki.contrib.jira.test.ui;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebElement;
+import org.xwiki.administration.test.po.AdministrationPage;
+import org.xwiki.ckeditor.test.po.CKEditor;
+import org.xwiki.contrib.jira.test.po.JIRAAdministrationSectionPage;
+import org.xwiki.model.reference.DocumentReference;
+import org.xwiki.model.reference.EntityReference;
+import org.xwiki.test.docker.junit5.UITest;
+import org.xwiki.test.ui.TestUtils;
+import org.xwiki.test.ui.po.ViewPage;
+import org.xwiki.test.ui.po.editor.WYSIWYGEditPage;
+import org.xwiki.test.ui.po.editor.WikiEditPage;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+/**
+ * Verify the functionality of the CKEditor paste plugin.
+ *
+ * @version $Id$
+ * @since 10.0
+ */
+@UITest
+public class PastePluginIT
+{
+    private static final String ADMIN_CONFIG_JIRA_ID = "testId";
+
+    private static final String ADMIN_CONFIG_JIRA_URL = "https://localhost:8889";
+
+    private static final EntityReference TEST_PAGE_REFERENCE = new DocumentReference("wiki", "Test", "Macro");
+
+    private static final String LINK_JIRA_TICKET = ADMIN_CONFIG_JIRA_URL + "/browse/XWIKI-1000";
+
+    private static final String LINK_JIRA_JQL = ADMIN_CONFIG_JIRA_URL + "/browse/JIRA-65?jql=project%20%3D%20JIRA"
+        + "%20AND%20resolution%20%3D%20Unresolved%20ORDER%20BY%20priority%20DESC%2C%20updated%20DESC";
```

Comment:
This URL looks weird since it specified JIRA-65 and a JQL query.

Reply:
From my experience, this seems to be the norm. Whenever I query for some issues, the link always specifies the id of some issue that is part of the jql result. I can drop the issue id for testing purposes, but I think having the id reflects real life more accurately.

Reply:
That's because you don't use a "clean" URL :)

You need to click the "share" button to share the URL for a query.

Reply:
Oh. I never knew thats a thing. Thank you for sharing! Ill update the code as necessary.